packages/multichain-transactions-controller/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.0]
```

Comment:
Nit: I am not sure we already want to go to v1 for this package ? What dot you think about making a minor release ?